tests/OpenCFP/Http/Controller/Admin/TalksControllerTests.php

```diff
@@ -0,0 +1,184 @@
+<?php
+namespace OpenCFP\Tests\Http\Controller\Admin;
+
+use Mockery as m;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\HttpFoundation\Session\Storage\MockFileSessionStorage;
+use OpenCFP\Application;
+use OpenCFP\Environment;
+
+class AdminTalkControllerTest extends \PHPUnit_Framework_TestCase
+{
+    public $app;
```

Comment:
I'm getting ready to merge this in. I've created https://github.com/opencfp/opencfp/issues/271 as an effort to make a sweep through on stuff like this.